README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
+
+1. Clone our project from github (remember about including --recursive flag due to our submodule(s))::
+
+    $ git clone --recursive https://github.com/StingraySoftware/stingray.git
+
+2. Go to directory where you cloned project into and install astropy_helpers submodule::
```

Comment:
To test this, I created a clean `conda` environment (Python 3.5), installed all dependencies, then ran `make html` in the `docs` directory and `python setup.py test`. Everything went smoothly.